.travis.yml

```diff
@@ -35,22 +35,33 @@ cache:
     - $HOME/.gradle/native
     - $HOME/.gradle/wrapper
 
-# Emulator Management: Create, Start and Wait
-before_script:
-  - echo no | android create avd --force -n test -t $ANDROID_TARGET --abi $ANDROID_ABI
-  - emulator -avd test -no-skin -no-audio -no-window &
+before_install:
+  # Install SDK license so Android Gradle plugin can install deps.
+  - mkdir "$ANDROID_HOME/licenses" || true
+  - echo "d56f5187479451eabf01fb78af6dfcb131a6481e" > "$ANDROID_HOME/licenses/android-sdk-license"
```

Comment:
Does this need to be an encrypted variable?

Reply:
it's the same for everyone

Reply:
👍